app/src/main/java/com/gophillygo/app/adapters/AttractionListAdapter.java

```diff
@@ -74,7 +82,10 @@ public void optionsButtonClick(View view, T attraction) {
         PopupMenu menu = new PopupMenu(context, view);
         menu.getMenuInflater().inflate(R.menu.place_options_menu, menu.getMenu());
         menu.setOnMenuItemClickListener(item -> {
-            Log.d(LOG_LABEL, "Clicked " + item.toString());
+            AttractionFlag flag = attraction.createAttractionFlag(item.getItemId());
+            attraction.setFlag(flag);
+            viewModel.updateAttractionFlag(flag);
```

Comment:
This seems like an action that should be able to be accomplished with one call to set the flag, or maybe two if the update to the `LiveData` source object isn't used.